internal/pkg/cli/flag.go

```diff
@@ -57,12 +61,17 @@ const (
 	appTypeFlagDescription = "Type of application to create."
 	profileFlagDescription = "Name of the profile."
 	yesFlagDescription     = "Skips confirmation prompt."
-	jsonFlagDescription    = "Output in JSON format."
+	jsonFlagDescription    = "Optional. Outputs in JSON format."
 
 	dockerFileFlagDescription        = "Path to the Dockerfile."
 	imageTagFlagDescription          = `Optional. The application's image tag.`
 	stackOutputDirFlagDescription    = "Optional. Writes the stack template and template configuration to a directory."
 	prodEnvFlagDescription           = "If the environment contains production services."
+	limitFlagDescription             = "Optional. The maximum number of log events returned."
+	followFlagDescription            = "Optional. Specifies if the logs should be streamed."
+	sinceFlagDescription             = "Optional. Shows logs newer than a relative duration. Example: 1s, 2m, or 3h. Cannot be used with --start-time."
```

Comment:
I find this description to be more intuitive: https://github.com/kubernetes/kubectl/blob/0555a86aaff9f0ba20dbc066dfaa1891222def87/pkg/cmd/logs/logs.go#L170